app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from cafe import Cafe
+from errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends, cafe):
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
The current logic returns immediately if any friend is not vaccinated, which means it won't check the rest of the friends. If the requirement is to ensure all friends are checked for mask compliance even if one is not vaccinated, consider accumulating results and returning after the loop.